js/BoxMenuGroupView.js

```diff
@@ -11,6 +11,17 @@ class BoxMenuGroupView extends MenuItemView {
     _.defer(this.addChildren.bind(this));
     this.$el.imageready(this.setReadyStatus.bind(this));
     this.$el.parents('.boxmenu__item-container').addClass('has-groups');
+    this.updateItemCount();
+  }
+
+  updateItemCount() {
+    const models = this.model.getChildren().models;
+    const totalChildren = models.length;
+    models.forEach(model => {
+      model.set({
+        _totalChild: totalChildren
+      });
+    });
```

Comment:
```suggestion
  updateItemCount() {
    const models = this.model.getChildren().models;
    const totalChildren = models.length;
    models.forEach(model => model.set('_totalChild', totalChildren));
```
With the fat arrow functions the block braces `{}` for the function body are optional, so if you have one line you don't need them.
The model.set function can take a `name, value` or `{ name1: value1, name2: value2 }` syntax. so we can further squish down.